packages/create-react-app-template-typescript/package.json

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "create-react-app-template-typescript",
```

Comment:
Do we need separate packages or could we keep the default template within `react-scripts`? i.e. not specifying `--template` gives you default js, `--template=typescript` gives you our TS template, otherwise all of the other options you've provided below? Would that simplify our release process?

Reply:
I understand what you're saying, and agree with the logic.

I feel that we should ship templates, as examples and also to ensure that we own those package names on NPM, but this would help with migration issues for sure.

We could do both for now, and then aim to move to templates-only for the next major? What do you think about that?

Reply:
Yeah that could work. I don't have a strong opinion either way.

Reply:
@iansu It would be great to get your opinion here too ;)